README.md

```diff
@@ -69,6 +69,10 @@ the benchmark's workflow is executed.
   $ kubectl create -n kubestone -f config/samples/perf_v1alpha1_iperf3.yaml
   iperf3.perf.kubestone.xridge.io/iperf3-sample created
   ```
+* Start fio benchmark by creating CR
+  ```bash
+  $ kustomize config/samples/fio/overlays/builtin_jobs | kubectl create -f -
```

Comment:
`kubestone` namespace is missing from here

Reply:
done